statusHandler/statusMetricsProvider.go

```diff
@@ -10,13 +10,14 @@ import (
 
 // statusMetrics will handle displaying at /node/details all metrics already collected for other status handlers
 type statusMetrics struct {
-	nodeMetrics *sync.Map
+	nodeMetrics   map[string]interface{}
```

Comment:
since we only have 3 types of values: strings, uint64 and int64 we could have had 3 different maps. We would have avoid all those casts (which consume CPU) and the memory footprint should have been almost the same

Reply:
done. also, each map has its own mutex